lib/msal-core/src/telemetry/UiEvent.ts

```diff
@@ -0,0 +1,19 @@
+import TelemetryEvent from "./TelemetryEvent";
+import { EVENT_NAME_PREFIX } from "./TelemetryConstants";
+
+const USER_CANCELLED = EVENT_NAME_PREFIX + "user_cancelled";
+const ACCESS_DENIED = EVENT_NAME_PREFIX + "access_denied";
+
+export default class UiEvent extends TelemetryEvent {
+    constructor(correlationId: string) {
+        super(`${EVENT_NAME_PREFIX}ui_event`, correlationId);
+    }
+
+    public set userCancelled(userCancelled: boolean) {
+        this.event[USER_CANCELLED] = userCancelled.toString();
+    }
+
+    public set accessDenied(accessDenied: boolean) {
```

Comment:
When does this event get triggered?

Reply:
opposite of user cancelled? the admin required or something?

Reply:
@sameerag doesn't the PR in #811 fix the issue that occurs when accessDenied is given? If this is something we are detecting for telemetry why aren't we handling it as an error in our library?

Reply:
@pkanher617 We can. For access_denied we may need to read the sub-error sent by the server team, apparently, their messaging is vague intentionally as a security smoke screen. I am following up with Adrian Frei on what would be an appropriate error messaging in this case. Will close loop once I have their response.

Reply:
if we do not see this from server that is fine, for now it is supported in the Event

Reply:
Ok just keep it in mind as we move forward.